source/request-as-event-emitter.js

```diff
@@ -47,13 +47,26 @@ module.exports = options => {
 
 		/* istanbul ignore next: electron.net is broken */
 		if (options.useElectronNet && process.versions.electron) {
-			const electron = global['require']('electron'); // eslint-disable-line dot-notation
+			const electron = require('electron');
 			fn = electron.net || electron.remote.net;
 		}
 
 		let timings;
 		const cacheableRequest = new CacheableRequest(fn.request, options.cache);
 		const cacheReq = cacheableRequest(options, response => {
+			if (options.useElectronNet) {
```

Comment:
Can you add a comment with a link to the Electron issue this fixes?

Reply:
Yeah, I'll do.